src/ctim/schemas/openc2_network.clj

```diff
@@ -0,0 +1,117 @@
+(ns ctia.schemas.openc2-network
+  (:require [schema.core :as s]
+            [ctia.schemas.openc2vocabularies :as openc2v]))
+
+
+
+
+(s/defschema BGPBlackhole
+  {:type (s/enum "BGPBlackhole")
+   :host s/Str
+   })
+
+(s/defschema DNSSinkhole
+  {:type (s/enum "DNSSinkhole")
+   :host s/Str
+   })
+
+(def Protocol
+  (s/enum "TCP"
+          "UDP"
+          "ICMP"
+          "Any"))
+
+(def ACLAction
+  (s/enum "ALERT" 
+          "DROP"
+          "DENY"
+          "LOG"
+          "PASS"
+          "REJECT"))
+
+(s/defschema TrafficType
+  {:protocol Protocol
+   :sourceAddress s/Str
+   :sourcePort s/Str
+   :destinationAddress s/Str
+   :destinationPort s/Str
+   })
+
+(s/defschema NetworkACL
+  {:type (s/enum "NetworkACL")
+   :traffic TrafficType
+   :action ACLAction
+   })
+
+
+(s/def vlanProfile
+  {:vlanTag s/Str})
+
+(s/def secGroupProfile
+  {:secGroupTag s/Str
+   :secGroupACL s/Str
+   })
+
+
+(s/defschema Remediation
+  {:type (s/enum "Remediation")
+   :server s/Str
+   :containmentProfile (s/either vlanProfile
+                                 secGroupProfile)
+   :acl NetworkACL 
+   })
+
+(s/defschema NonSensitive
+  {:type (s/enum "NonSensitive")
+   :permissibleIps [s/Str]
+   :acl NetworkACL 
+   })
+
+(s/defschema Honeypot
+  {:type (s/enum "Honeypot")
+   :permissibleIps [s/Str]
+   :acl NetworkACL 
+   :routes {:prefix s/Str
+            :NextHop s/Str
+            :NextHopType s/Str}
+   })
+
+(def EncapsulationType
+  (s/enum "GRE"
+          "VXLAN"))
+
+(s/defschema BlockModifierType
```

Comment:
I think we can dispense with the use of the term "Type" throughout all of our schema names--we did so through the rest of the CTIM, so this would just be:

``` clj
(defschema BlockModifier
  ...
)
```